Exercícios JS

VictorMagosso opened the pull request.

Commit:
Exercicios IF

Commit:
Exercicios WHILE_IF

3 e 4 incompletos ainda

Commit:
Exercicio 5 acrescentando for para resolução

davipanico reviewed (changes requested):
Fala @VictorMagosso tranquilo? O arquivo 'E5' da lista_4 está vazio, não posso dar merge dessa forma! Consegue corrigir por favor? Fico no aguardo !

VictorMagosso committed:
Era apenas um teste

Estava treinando com o Johnatan fazer o commit por linha de comando no Git BASH... acabei esquecendo de removê-lo

VictorMagosso commented:
Opa, tudo bem?

Ah sim, o johnatan tava me ensinando a fazer o commit pelo git bash e acabei fazendo esse como teste mas não deletei. Deletei agora, você consegue ver a atualização?

Obrigado!!

Get Outlook for Android<https://aka.ms/ghei36>

________________________________
From: Davi Moreira <notifications@github.com>
Sent: Tuesday, July 7, 2020 10:57:23 AM
To: arbyte-br/Arbyte-Atividades <Arbyte-Atividades@noreply.github.com>
Cc: Victor Magosso <victor_magosso@hotmail.com>; Mention <mention@noreply.github.com>
Subject: Re: [arbyte-br/Arbyte-Atividades] Exercícios JS (#125)


Merged #125<https://github.com/arbyte-br/Arbyte-Atividades/pull/125> into master.

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub<https://github.com/arbyte-br/Arbyte-Atividades/pull/125#event-3520574063>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/APAFXZYDDT32PWEI63OMAQDR2MSUHANCNFSM4OR6UOKQ>.
